requirements.txt

```diff
@@ -1,8 +1,4 @@
-# Replace the following with something like dwave-ocean-sdk>=6.4.2
-dimod==0.12.12
-dwave-networkx==0.8.14
-dwave-samplers==1.0.0
-dwave-system==1.19.0
+dwave-ocean-sdk>=6.6.0
```

Comment:
It doesn't really need to be as high as 6.6, right?

Reply:
It needs [dimod 12.12](https://github.com/dwavesystems/dwave-ocean-sdk/commit/75e3b2eef75efab4d3d2143c8514efd72e3517f8) for the wireless generators

Reply:
So it's ocean 5.5. Ok, that's pretty close.

Reply:
> So it's ocean 5.5. Ok, that's pretty close.

I think you mean 6.5, then yes